dts/bindings/display/galaxycore,gc9x01x.yaml

```diff
@@ -0,0 +1,158 @@
+# Copyright (c) 2023 Mr Beam Lasers GmbH.
+# Copyright (c) 2023 Amrith Venkat Kesavamoorthi <amrith@mr-beam.org>
+# Copyright (c) 2023 Martin Kiepfer <mrmarteng@teleschirm.org>
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  GC9X01X display driver.
+
+  This driver implements support for various GC9xxx graphics
+  controllers and different display sizes. It has been validated
+  for following controllers:
+   - GC9101A: (Waveshare 240x240, 1.28inch round lcd display 240x240)
+
+  Here is an example to define a display interface:
+
+  / {
+    pwmleds {
+      compatible = "pwm-leds";
+      backlight_pwm: backlight_pwm {
+        pwms = <&ledc0 0 PWM_MSEC(10) PWM_POLARITY_NORMAL>;
+      };
+    };
+  };
```

Comment:
one more: this is now outdated

Reply:
Thought we want to leave this in for reference.
Fixed in [60c20cb](https://github.com/zephyrproject-rtos/zephyr/commit/60c20cb0e4f999ebe39a5e27b1f7fae6c01a5df4)